src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -56,17 +56,17 @@
   <ItemGroup>
     <PackageReference Include="log4net" Version="2.0.15" />
     <PackageReference Include="Microsoft.AspNetCore.OData" Version="7.5.12" />
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="3.1.5" />
```

Comment:
For test dependencies vulnerabilities, I have upgraded to the lowest fixed minor but highest patch of that minor.